.github/workflows/ci.yml

```diff
@@ -12,7 +12,7 @@ on:
 
 jobs:
   test:
-    uses: fastify/workflows/.github/workflows/plugins-ci.yml@v3
+    uses: fastify/workflows/.github/workflows/plugins-ci.yml@v4
```

Comment:
This is a breaking change, as v4 drops CI testing for node 14 and 16.

Reply:
Implemented skipping test suites so it exits with 0 on older node versions.
Other cli related tests are still being ran on those.